packages/aws-cdk/lib/version.ts

```diff
@@ -1,17 +1,21 @@
 import { exec as _exec } from 'child_process';
 import colors = require('colors/safe');
-import { close as _close, open as _open, stat as _stat } from 'fs';
+import fs = require('fs');
+import { mkdirsSync } from 'fs-extra';
+import os = require('os');
+import path = require('path');
 import semver = require('semver');
 import { promisify } from 'util';
 import { debug, print, warning } from '../lib/logging';
 import { formatAsBanner } from '../lib/util/console-formatters';
 
 const ONE_DAY_IN_SECONDS = 1 * 24 * 60 * 60;
 
-const close = promisify(_close);
+const close = promisify(fs.close);
```

Comment:
This is exactly what's no longer necessary with `fs-extra`.

Reply:
Yep. This got much simpler.